lib/stripe/list.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Stripe.List do
```

Comment:
Mostly because we will want to be able to fetch more pages eventually. This will probably be a function of this struct.